explainaboard/analysis/feature_funcs.py

```diff
@@ -1,14 +1,34 @@
 from __future__ import annotations
 
 from collections.abc import Callable, Iterator
-from typing import Any
 
 from lexicalrichness import LexicalRichness
 import sacrebleu
 
+from explainaboard.info import SysOutputInfo
 from explainaboard.utils import basic_words
 from explainaboard.utils.logging import progress
 from explainaboard.utils.tokenizer import Tokenizer
+from explainaboard.utils.typing_utils import unwrap
+
+
+def _get_tokens(
+    sys_info: SysOutputInfo, text: str | list[str], side: str = 'source'
+) -> list[str]:
+    if isinstance(text, list):
+        if side != 'none':
```

Comment:
Is there any reason to take care about the value of `side` here? If no, simply ignoring this value looks better.